app/Bus/Commands/User/AddTeamMemberCommand.php

```diff
@@ -48,8 +48,9 @@
      */
     public $rules = [
         'name'     => 'required|string',
-        'password' => 'string',
-        'level'    => 'int',
+        'email'    => 'required|email',
+        'password' => 'required|string',
+        'level'    => 'int|min:1|max:2',
```

Comment:
In all our rules on L5.3, if they're not required, we need to add `nullable` to them.